base_import_match/models/base_import.py

```diff
@@ -88,7 +89,7 @@ def _compute_name(self):
                 " + ".join(
                     s.field_ids.mapped(
                         lambda r: (
-                            str(r.field_id.name) +
+                            unicode(r.field_id.name) +
```

Comment:
odoo return a unicode string here then we don't need a `unicode(string_unicode)` again.
The problem is use `"{}".format()` instead of `u"{}".format()` (Notice the `u` prefix)